apps/omg_watcher_rpc/mix.exs

```diff
@@ -0,0 +1,55 @@
+defmodule OMG.WatcherRPC.Mixfile do
+  use Mix.Project
+
+  def project do
+    [
+      app: :omg_watcher_rpc,
+      version: OMG.Umbrella.MixProject.umbrella_version(),
+      build_path: "../../_build",
+      config_path: "../../config/config.exs",
+      deps_path: "../../deps",
+      lockfile: "../../mix.lock",
+      elixir: "~> 1.4",
+      elixirc_paths: elixirc_paths(Mix.env()),
+      compilers: [:phoenix] ++ Mix.compilers(),
+      start_permanent: Mix.env() == :prod,
+      aliases: aliases(),
+      deps: deps(),
+      test_coverage: [tool: ExCoveralls]
+    ]
+  end
+
+  def application do
+    [
+      mod: {OMG.WatcherRPC.Application, []},
+      extra_applications: [:logger, :runtime_tools]
+    ]
+  end
+
+  # Specifies which paths to compile per environment.
+  defp elixirc_paths(:prod), do: ["lib"]
+  defp elixirc_paths(_), do: ["lib", "test/support"]
+
+  defp deps do
+    [
+      {:phoenix, "~> 1.3"},
+      {:plug_cowboy, "~> 1.0"},
+      {:phoenix_ecto, "~> 4.0"},
+      {:postgrex, "~> 0.14"},
+      {:ecto_sql, "~> 3.1"},
```

Comment:
Do we need really these 3: `phoenix_ecto, postgrex, ecto_sql`
DB is used in Watcher, and RPC layer should call only Watcher's API layer.